index.html

```diff
@@ -1,246 +1,139 @@
+                <li class="protitle"><h3>Intro Call</h3></li>
+                <li class="protext">Met CEO Ariel Online, discussing about all possibilities which come with joining Microverse.
+                </li>
+            </ul>
+
+            <ul class="procard">
+                <li class="proimg">
+                    <i class="comments icon"></i>
+                </li>
+                <li class="protitle"><h3>Forum</h3></li>
+                <li class="protext">Share your thoughts and opinions with student success about what's next in version 2.
+                </li>
+            </ul>
+            <ul class="procard">
+                <li class="proimg">
+                    <i class="wait icon"></i>
+                </li>
+                <li class="protitle"><h3>Program Hours</h3></li>
+                <li class="protext">After joining the program you have to dedicate 8+ hours of learning each day.
+                </li>
+            </ul>
+            <ul class="procard">
+                <li class="proimg">
+                    <i class="users icon"></i>
+                </li>
+                <li class="protitle"><h3>Workshop</h3></li>
+                <li class="protext">Complete a project with Microverse team and learn how Microverse change lives.
+                </li>
+            </ul>
         </div>
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
+        <a href="#" class="promore">SEE THE WHOLE PROGRAM</a>
+        <button type="button" class="probtn">Join Microverse</button>
+    </section>
 
-          <div class="feature-card">
-            <h3>Lecturer3</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
-        <div class="card-lec">
-          <img src="./images/Picture3.png" alt="lecture" />
-          <div class="feature-card">
-            <h3>Lecturer</h3>
-            <h5>Speaker</h5>
-            <p>
-              Listen to the speaker from various countries about sharing and
-              opening
-            </p>
-          </div>
-        </div>
+    <!--Added with Javscript Dynamically-->
+    <section class="featuredspeakers">
+      <h2 class="speakerhead section-title">Featured Speakers</h2>
+    </section>
 
-        <div class="dis-none">
-          <button class="show-less" type="button">LESS &#x22C0;</button>
+    <footer class="footer">
+        <div class="partners">
+            <h2 class="partnerhead section-title">Partner</h2>
+            <div class="partnerlogo">
+                <img class="plogo" src="img/tech.png" alt="Tech Crunch logo">
+                <img class="plogo" src="img/EdSurgelogo.png" alt="EdSurge logo">
+                <img class="plogo" src="img/CincoDíaslogo.png" alt="Cinco Días logo">
+                <img class="plogo" src="img/Infobaelogo.png" alt="Infobae logo">
```

Comment:
- [ ] [Optional] [Highly Recommended] For the footer in the mobile view It would be better if you increase the width of the footer to make it looks good because now it's smooshed and not looking so great.

Reply:
ok thank you so much God bless you, I will make recommended change also